src/controller/CommissioneeDeviceProxy.h

```diff
@@ -196,6 +196,14 @@ class CommissioneeDeviceProxy : public DeviceProxy, public SessionReleaseDelegat
 
     void SetActive(bool active) { mActive = active; }
 
+    /**
+     * @brief
+     * Should be called when the pairing completes.
```

Comment:
Called by whom? If it's called externally (e.g. no by `onSessionEstablished`), maybe mention it has to be called in a safe context relative to the CHIP stack.

Also suggest saying that the return values map to `LoadSecureSessionParametersIfNeeded`.

Reply:
Is that not an implementation detail? Maybe we should remove the "should be called" and just state what it does.

Reply:
Changed to indicate what it does.